src/views/modal/item-total.php

```diff
@@ -11,12 +11,17 @@
  * @since TBD
  *
  * @version TBD
- * @todo Adjust with #133179
  */
+
+$ticket = $this->get( 'ticket' );
+
+/** @var Tribe__Tickets__Commerce__Currency $currency */
+$currency        = tribe( 'tickets.commerce.currency' );
```

Comment:
Is there anything we can do to move this out of the view? Can it be provided like the other data to the view since it's useful for other views/contexts?